website/models.py

```diff
@@ -238,8 +237,8 @@ def hostname_domain(self):
     @property
     def domain_name(self):
         parsed_url = urlparse(self.url)
-        domain = parsed_url.hostname
```

Comment:
curious as to why this was changed?

Reply:
@AtmegaBuzz can you comment on why this changed?

Reply:
@AtmegaBuzz this is meant to extract the domain name from the url that was reported, it looks like it was operating correctly.  Can you please revert this?  If there is a specific case where it has to change, can you please add an automated test to show that it's working as intended?

Reply:
@AtmegaBuzz please revert this unless you add a test to make sure it works